xdsl/dialects/snitch_runtime.py

```diff
@@ -136,34 +124,46 @@ def __init__(
         )
 
 
+Dma1DWideptrOp = DmaStart1DBaseOp[Annotated[Attribute, u64]]
+Dma1DOp = DmaStart1DBaseOp[Annotated[Attribute, u32]]
+Dma2DWideptrOp = DmaStart2DBaseOp[Annotated[Attribute, u64]]
+Dma2DOp = DmaStart2DBaseOp[Annotated[Attribute, u32]]
```

Comment:
Do you need to define these aliases here? I think you should be able to just put these expressions in the parenthesis of the classes that use them?

Reply:
That would look a lot better indeed, I'll fix it!